packages/devextreme-cli/src/templates/react/application/src/components/change-password-form/ChangePasswordForm.tsx

```diff
@@ -58,6 +58,8 @@ export default function ChangePasswordForm() {
           <RequiredRule message="Password is required" />
           <CustomRule
             message={'Passwords do not match'}
+            // eslint-disable-next-line
+            // @ts-ignore
```

Comment:
Get rid of ts-ignore

Reply:
@iBat was it required?

Reply:
This is where the TS linter error is without ts-ignore.

Reply:
Okay, I will fix the cause of the fail